EIPS/eip-4337.md

```diff
@@ -0,0 +1,313 @@
+---
+eip: 4337
+title: Account Abstraction via Entry Point Contract specification
+author: Vitalik Buterin, Yoav Weiss, Kristof Gazso, Namra Patel, Dror Tirosh
+discussions-to: https://ethereum-magicians.org/t/erc-4337-account-abstraction-via-entry-point-contract-specification/7160
+type: Standards Track
+category: ERC
+status: Draft
+created: 2021-09-29
+---
+
+## Simple Summary
```

Comment:
This section has been deprecated in favor of the `description` field in the preamble. You can see how in should be structured in the [`eip-template.md`](https://github.com/ethereum/EIPs/blob/master/eip-template.md).